Review 2 (#8)
CTF-handbooks/map-maker-handbook: HANDBOOK.md

  
***
### 4. Setting up
* Grant yourself the "ctf_map_editor" priv by running `/grantme ctf_map_editor`. You can also use `/grantme all` to grant yourself "ctf_map_editor" and all the other privs that will be useful while making a map, such as fly, noclip, fast, etc.
* You can hit <kbd>Esc</kbd> on your keyboard and click `Change Keys` to see your controls (such as how to fly, noclip, etc.).

  ![image](https://github.com/CTF-handbooks/map-maker-handbook/assets/88883098/98117763-d42c-4a64-ade6-a894c14f61a9)

> * *Tip*: Run `/time 12000` and `/set -n time_speed 0` if you want to disable nighttime while building the map.

Reviewer: ```suggestion
* *Tip*: run `/time 12000` and `/set -n time_speed 0` if you want to disable nighttime while building the map.
```
I don't think upper case is needed here since "Tips" starts with an upper case letter.

  ```
  //mtschemplace map_terrain
  ```

> * You should place your terrain there. If you're unhappy with the result, you can return, pick another landscape, and do it again.

Reviewer: ```suggestion
* With that, your terrain should be placed there successfully. If you're unhappy with the result, you can go back down, pick another terrain, and do it again.
```
Since it is already placed at this point, I don't think this makes sense.
Terrain is used previously, so why are we using landscape now?
"Return" seems a bit vague, but there are probably better options.

  
  ![image](https://github.com/CTF-handbooks/map-maker-handbook/assets/88883098/b57294e1-b6f1-4a5d-aeb7-40408c5c580d)

* Follow the instructions given to you in chat and select the area your map is in, just like how you would select the area with WorldEdit. You can either punch a node to select or stand where you want to select and run the following command:
  ```
  /ctf_map here
  ```
  ![image](https://github.com/CTF-handbooks/map-maker-handbook/assets/88883098/bdca51e0-a439-4bf3-a65a-75530f2f5d2d)
  
  Make sure you selected your entire map and make sure you did not select any extra space. If you have decided to leave some terrain/structures outside your outer glass barrier, include them too.
***
### 11. Exporting the map
* After selecting your map area, run `/ctf_map editor` in chat again. This will open the Map Editor.

> * Check `Map Enabled` unless you don't want to make the map available for play. If you are making the map for addition to the official server, you will check this.

Reviewer: ```suggestion
* Check `Map Enabled` unless you don't want to make the map available for play. If you are making the map for the official server, you will check this.
```
Maybe this is slightly less confusing.

Author: Yeah

  
  ![image](https://github.com/CTF-handbooks/map-maker-handbook/assets/88883098/a1bfc7bb-0b84-4cea-b41c-b15668c15f55)

* In the license field, add the license of your map. Every map must have a license. The official CTF server will only accept free culture licenses like `CC BY-SA 4.0`. Please note that some CC licenses, such as `CC BY-ND` are not free. You can find a list of CC licenses [here](https://creativecommons.org/share-your-work/cclicenses/).

  ![image](https://github.com/CTF-handbooks/map-maker-handbook/assets/88883098/e3e0a9e8-8968-423a-8b54-3288f303d0d9)

  From the ctf_map README:

>   > You can append any attribution you need to give to the `license` field (For example: If you modified someone's map or used one of their builds, you'd list their name and what map/build of theirs that you modified/used). If you want to give more information, you can use the `Other Info` field.

Reviewer: ```suggestion
  > You can append any attribution you need to give to the `license` field (For example: If you modified someone's map or used one of their builds you'd list their name and what map/build of theirs that you modified/used). If you want to give more information, you can use the `Other Info` field.
```
This is a straight out quote from the ctf_map README, so I'm not editing.

Author: Ah right, it is from the quote.

  ![image](https://github.com/CTF-handbooks/map-maker-handbook/assets/88883098/97132307-260e-4d1a-b065-f909d30ee39b)

* From the ctf_map README:
  > `initial_stuff` are the items given to players at their (re)spawn. The `initial_stuff` field is located in the `map.conf` file. At least a pickaxe and some torches should be given in the map's `initial_stuff`.

>   > Below is an example of `initial_stuff` value that registers a stone pickaxe, 30 cobblestones, 5 torches and a pistol.

Reviewer: ```suggestion
  > An example of `initial_stuff` value that registers a stone pickaxe, 30 cobblestones, 5 torches and a pistol is given below.
```
This is a straight out quote from the ctf_map README, so I'm not editing unless the original text was edited.

   * `2000` is for 2 AM
   * etc.
  
* Below the map constants, you will see a list of the different teams with a checkbox next to them. Use the checkbox to enable the teams that exist in your map.
  
  ![image](https://github.com/CTF-handbooks/map-maker-handbook/assets/88883098/6b893f4d-ef85-460c-a67a-a2b60eb2f49b)

  Next, click the `Set Flag Pos` button below your first enabled team and punch the node where that team's flag will be on. This will place the flag on top of that node.

>   Then, click the `Zone Bounds` button below the `Set Flag Pos` button and select the team's team-zone. A team-zone is where the team's players interact during build-time before the build-time barriers drop. During build-time, players who leave the team-zone will be teleported back to base. Even standing on the edge of the team-zone will teleport the player back to base, **so make sure to include the barriers within your team-zone too, or players will get teleported back to base if they go through the barrier area during build-time!**

Reviewer: ```suggestion
  Then, click the `Zone Bounds` button below the `Set Flag Pos` button and select the team's team-zone. A team-zone is the area where the team's players interact within during build-time before the build-time barriers drop. During build-time, players who leave the team-zone will be teleported back to base. Even standing on the edge of the team-zone will teleport the player back to base, **so make sure to include the barriers within your team-zone too, or players will get teleported back to base if they touch the barriers during build-time!**
```
Bit unclear.
For the second bit, yes, I meant "touch the barriers".

Author: Oh, I misunderstood the sentence as the barriers should teleport. My bad

***
### 12. Testing the map
* You can find your exported map in `[Minetest folder]/worlds/[Map World]/schems/`.

> * To test your map, copy that folder into `[Minetest folder]/games/capturetheflag/mods/ctf/ctf_map/maps`. You can go to your Minetest folder and paste the following into your file manager's search to navigate there quickly:

Reviewer: ```suggestion
* To test your map, copy that folder into `[Minetest folder]/games/capturetheflag/mods/ctf/ctf_map/maps`. You can go to your Minetest folder and paste the following into your file manager's search/directory to navigate there quickly:
```
Not search in certain file apps.

Author: Hmm, ok.

  ```

>   games/capturetheflag/mods/ctf/ctf_map/maps

Reviewer: ```suggestion
  /games/capturetheflag/mods/ctf/ctf_map/maps
```
Needs "/" on certain file apps. If not needed just copy it without selecting the first "/"

Author: On Nautilus and Thunar it doesn't from my testing. How would a snippet below mentioning it be?

Reviewer: Sure.

  ```
  /ctf_teams set singleplayer blue
  ```

***
### 13. Screenshot
From the ctf_map README:
> If you choose to submit your map, include a screenshot of it in the exported map's folder. It should be taken without any texture packs enabled and must have an aspect ratio of 3:2 (screenshot `600px`x`400px` is suggested).

> You can take a screenshot easily by doing the following:

> > 1. Hide the HUD. By default, <kbd>F1</kbd> does that.

Reviewer: ```suggestion
> 1. Hide the HUD. By default <kbd>F1</kbd> does that.
```
Quote, not editing.

> > 2. Hide the chat log. By default, <kbd>F2</kbd> does that.

Reviewer: ```suggestion
> 2. Hide the chat log. By default <kbd>F2</kbd> does that.
```
Quote, not editing.
